sites/docs/content/components/dropdown-menu.md

```diff
@@ -16,4 +16,202 @@ description: Displays a menu of items that users can select from when triggered.
 
 </ComponentPreviewV2>
 
+## Structure
+
+```svelte
+<script lang="ts">
+	import { DropdownMenu } from "bits-ui";
+</script>
+
+<DropdownMenu.Root>
+	<DropdownMenu.Trigger />
+
+	<DropdownMenu.Content>
+		<DropdownMenu.Label />
+		<DropdownMenu.Item />
+
+		<DropdownMenu.Group>
+			<DropdownMenu.Item />
+		</DropdownMenu.Group>
+
+		<DropdownMenu.CheckboxItem>
+			<DropdownMenu.CheckboxIndicator />
+		</DropdownMenu.CheckboxItem>
+
+		<DropdownMenu.RadioGroup>
+			<DropdownMenu.RadioItem>
+				<DropdownMenu.RadioIndicator />
+			</DropdownMenu.RadioItem>
+		</DropdownMenu.RadioGroup>
+
+		<DropdownMenu.Sub>
+			<DropdownMenu.SubTrigger />
+			<DropdownMenu.SubContent />
+		</DropdownMenu.Sub>
+
+		<DropdownMenu.Separator />
+		<DropdownMenu.Arrow />
+	</DropdownMenu.Content>
+</DropdownMenu.Root>
+```
+
+## Reusable Components
+
+If you're planning to use Dropdown Menu in multiple places, you can create a reusable component that wraps the Dropdown Menu component.
+
+This example shows you how to create a Dropdown Menu component that accepts a few custom props that make it more capable.
+
+```svelte title="CustomDropdownMenu.svelte"
+<script lang="ts">
+	import type { Snippet } from "svelte";
+	import { DropdownMenu, type WithoutChild } from "bits-ui";
+
+	type Props = DropdownMenu.Props & {
+		buttonText: string;
+		items: { label: string; value: string }[];
+		contentProps?: WithoutChild<DropdownMenu.Content.Props>;
+		// other component props if needed
+	};
+
+	let {
+		open = $bindable(false),
+		children,
+		buttonText,
+		items,
+		contentProps,
+		...restProps
+	}: Props = $props();
+</script>
+
+<DropdownMenu.Root bind:open {...restProps}>
+	<DropdownMenu.Trigger>
+		{buttonText}
+	</DropdownMenu.Trigger>
+	<DropdownMenu.Content {...contentProps}>
+		<DropdownMenu.Group>
+			{#each items as item}
+				<DropdownMenu.Item textValue={item.value}>
```

Comment:
In this case `textValue === item.label`. `textValue` is only necessary if you have some complex content within the item and is only used for typeahead purposes

Reply:
I gave this a minute, and since there is no kind of `search` functionality going on here, wouldn't it be better to forego the `label`/`value` duo and just pass the values themselves. WDYT?